src/app/page/side-nav/side-nav.component.html

```diff
@@ -16,6 +16,7 @@
       <li><a routerLink="showcase/button" [routerLinkActive]="'is-selected'" class="side-nav__item">Button</a></li>
       <li><a routerLink="showcase/card" [routerLinkActive]="'is-selected'" class="side-nav__item">Card</a></li>
       <li><a routerLink="showcase/list" [routerLinkActive]="'is-selected'" class="side-nav__item">List</a></li>
+      <li><a routerLink="showcase/list-load-on-demand" [routerLinkActive]="'is-selected'" class="side-nav__item">List - load on demand</a></li>
```

Comment:
```suggestion
      <li><a routerLink="showcase/list/load-on-demand" [routerLinkActive]="'is-selected'" class="side-nav__item">List - load on demand</a></li>
```

Reply:
As we talked about, the menu styling does not support subroutes as we speak, and i don't want to makes changes for that in this PR.